packages/design-system/src/components/N8nTag/Tag.vue

```diff
@@ -0,0 +1,25 @@
+<template functional>
+	<div :class="$style.tag" v-text="props.tag" />
+</template>
+
+<script lang="ts">
+export default {
+	name: 'n8n-tag',
+	props: {
+		tag: {
+			type: String,
+		},
+	},
+};
+</script>
+
+<style lang="scss" module>
+.tag {
+	width: fit-content;
+	margin-right: 4px;
+	padding: 3px 4px;
+	background-color: #dbdfe7;
+	border-radius: var(--border-radius-base);
+	font-size: 11px;
```

Comment:
please use css variables wherever you can

Reply:
Done & updated  :) Only for this background-color and font-size: 11px; we do not have css variables.